pkg/trtl/metrics/metrics.go

```diff
@@ -132,36 +133,42 @@ func initMetrics() {
 		Namespace: PmNamespace,
 		Name:      "sync_latency",
 		Help:      "total duration of anti-entropy (originator perspective), labeled by peer and region",
+		Buckets:   prometheus.LinearBuckets(100, 200, 50),
```

Comment:
I think we might want to start the bins at 10 for sync_latency, and with a smaller step, perhaps
```golang
prometheus.LinearBuckets(10, 10, 50)
```